scipy/sparse/tests/test_base.py

```diff
@@ -47,6 +47,7 @@ class for generic tests" section.
 from scipy._lib.decorator import decorator
 
 import nose
+from nose.tools import assert_in
```

Comment:
This causes Travis to fail with Python 2.6. Not sure which Node version it's using, but it's easy to work around:

```py
try:
    from nose.tools import assert_in
except ImportError:
    def assert_in(x, collection):
        assert_(x in collection)
```